docs/getting-started-guides/mesos.md

```diff
@@ -0,0 +1,303 @@
+{ "kind": "Pod",
+"apiVersion": "v1beta1",
+"id": "nginx-id-01",
+"desiredState": {
+  "manifest": {
+    "version": "v1beta1",
+    "containers": [{
+      "name": "nginx-01",
+      "image": "dockerfile/nginx",
+      "ports": [{
+        "containerPort": 80,
+        "hostPort": 31000
+      }],
+      "livenessProbe": {
+        "enabled": true,
+        "type": "http",
+        "initialDelaySeconds": 30,
+        "httpGet": {
+          "path": "/index.html",
+          "port": "8081"
+        }
+      }
+    }]
+  }
+},
+"labels": {
+  "name": "foo"
+} }
+EOPOD
+```
+
+Send the pod description to Kubernetes using the `kubectl` CLI:
+
+```bash
+$ bin/kubectl create -f nginx.json
+nginx-id-01
+```
+
+Wait a minute or two while `dockerd` downloads the image layers from the internet.
+We can use the `kubectl` interface to monitor the status of our pod:
+
+```bash
+$ bin/kubectl get pods
+POD          IP           CONTAINER(S)  IMAGE(S)          HOST                       LABELS                STATUS
+nginx-id-01  172.17.5.27  nginx-01      dockerfile/nginx  10.72.72.178/10.72.72.178  cluster=gce,name=foo  Running
+```
+
+Verify that the pod task is running in the Mesos web console.  Click on the Kubernetes framework.  The next screen should show the running Mesos task that started the Kubernetes pod.
+
+## Run the Example Guestbook App
+
+Following the instructions from the kubernetes-mesos [examples/guestbook][7]:
+
+```bash
+$ export ex=k8sm/examples/guestbook
+$ bin/kubectl create -f $ex/redis-master.json
+$ bin/kubectl create -f $ex/redis-master-service.json
+$ bin/kubectl create -f $ex/redis-slave-controller.json
+$ bin/kubectl create -f $ex/redis-slave-service.json
+$ bin/kubectl create -f $ex/frontend-controller.json
+
+$ cat <<EOS >/tmp/frontend-service
+{
+  "id": "frontend",
+  "kind": "Service",
+  "apiVersion": "v1beta1",
+  "port": 9998,
+  "selector": {
+    "name": "frontend"
+  },
+  "publicIPs": [
+    "${servicehost}"
+  ]
+}
+EOS
+$ bin/kubectl create -f /tmp/frontend-service
+```
+
+Watch your pods transition from `Pending` to `Running`:
+
+```bash
+$ watch 'bin/kubectl get pods'
+```
+
+Review your Mesos cluster's tasks:
+
+```bash
+$ mesos ps
+   TIME   STATE    RSS     CPU    %MEM  COMMAND USER ID
+ 0:00:03    R    67.62 MB  0.5   105.66   none  root c7315c47-7a61-11e4-8b8c-42010a863922
+ 0:00:03    R    68.20 MB  0.75  106.57   none  root c731613c-7a61-11e4-8b8c-42010a863922
+ 0:00:01    R    67.79 MB  0.25  105.93   none  root c7310460-7a61-11e4-8b8c-42010a863922
+ 0:00:03    R    67.62 MB  0.5   105.66   none  root bda74154-7a61-11e4-8b8c-42010a863922
+ 0:00:03    R    68.20 MB  0.75  106.57   none  root bda6e97f-7a61-11e4-8b8c-42010a863922
+ 0:00:03    R    68.20 MB  0.75  106.57   none  root b0e206f1-7a61-11e4-8b8c-42010a863922
+```
+The number of Kubernetes pods listed earlier (from `bin/kubectl get pods`) should equal to the number active Mesos tasks listed the previous listing (`mesos ps`).
```

Comment:
FWIW I had 7 pods in 'kubectl get pods' and 'mesos ps' (3 x frontend-controller, 1 nginx, 1 redis-master, 2 x redis-slave-controller)